docs/training/apis-howto/api-core-ug.rst

```diff
@@ -1,212 +1,302 @@
 
--  metrics tracking
--  checkpoint tracking and preemption support
--  hyperparameter search
--  distributing work across multiple GPUs and/or nodes
+************
+ Objectives
+************
 
-These are the same features provided by the higher-level PyTorchTrial, DeepSpeedTrial, and
-TFKerasTrial APIs (the higher-level APIs are implemented using the Core API).
+These step-by-step instructions walk you through modifying a script for the purpose of performing
+the following functions:
 
-This user guide shows you how to get started using the Core API.
+-  Report metrics
+-  Report checkpoints
+-  Perform a hyperparameter search
+-  Perform distributed training
 
-.. _core-getting-started:
+After completing the steps in this user guide, you will be able to do the following:
 
-*****************
- Getting Started
-*****************
+-  Understand the minimum requirements for running an experiment
+-  Modify a script and an experiment configuration file
+-  Understand how to convert model code
+-  Use the Core API to train a model
 
-As a simple introduction, this example training script increments a single integer in a loop,
-instead of training a model with machine learning. The changes shown for the example model should be
-similar to the changes you make in your actual model.
+***************
+ Prerequisites
+***************
 
-The ``0_start.py`` training script used in this example contains your simple "model":
+**Required**
 
-.. literalinclude:: ../../../examples/tutorials/core_api/0_start.py
-   :language: python
-   :start-at: import
-
-To run this script, create a configuration file with at least the following values:
+-  A Determined cluster
 
-.. literalinclude:: ../../../examples/tutorials/core_api/0_start.yaml
-   :language: yaml
+**Recommended**
 
-The actual configuration file can have any name, but this example uses ``0_start.yaml``.
+-  :ref:`qs-mdldev`
 
-Run the code using the command:
+*****************************************************
+ Step 1: Get the Tutorial Files & Run the Experiment
+*****************************************************
 
-.. code:: bash
+To run an experiment, you need, at minimum, a script and an experiment configuration (YAML) file.
 
-   det e create 0_start.yaml . -f
+Create a new directory.
 
-If you navigate to this experiment in the WebUI no metrics are displayed because you have not yet
-reported them to the master using the Core API.
+Get the files from the :download:`core_api_pytorch_mnist.tgz </examples/core_api_pytorch_mnist.tgz>`
+download or from the `Github repository
+<https://github.com/determined-ai/determined/tree/master/examples/tutorials/core_api_pytorch_mnist>`_.
 
-.. _core-metrics:
+.. note::
 
-****************
- Report Metrics
-****************
+   Throughout this user guide, we’ll show you how to make modifications to your script. After each
+   step, we’ll re-run our experiment using the appropriate script and its accompanying experiment
+   configuration file.
```

Comment:
> we’ll show you how to make modifications to your script

Not sure if this is worth rewording, but it sounds like they'll have to make modifications to a starting script to complete the tutorial, while in real life we actually have all stages pre-written in the example tgz file.

Reply:
working on this

Reply:
resolved